scripts/superdesk/editor2/editor.js

```diff
@@ -1151,8 +1151,14 @@ angular.module('superdesk.editor2', [
 
                         spellcheck.suggest(node.textContent).then(function(suggestions) {
                             if (scope.sentenceWord) {
-                                suggestions.push(scope.replaceWord[0].toUpperCase() + scope.replaceWord.slice(1));
-                                scope.suggestions = _.unique(suggestions);
+                                suggestions.push({
+                                    key: scope.replaceWord[0].toUpperCase() + scope.replaceWord.slice(1),
+                                    value: scope.replaceWord[0].toUpperCase() + scope.replaceWord.slice(1)
+                                });
+
+                                scope.suggestions = suggestions.filter(suggestion => {
+                                    return suggestion.key !== scope.replaceWord;
```

Comment:
You don't necesarilly have to change this, as it is already correct, but just FYI, `return` can be omitted here with this form:
```js
scope.suggestions = suggestions.filter(suggestion => suggestion.key !== scope.replaceWord);
```